Tiltfile

```diff
@@ -0,0 +1,5 @@
+
+# If not using a standard local dev name, specify your k8s context here
+#allow_k8s_contexts('jenkins-dev')
+k8s_yaml(helm('charts/jenkins', values='charts/jenkins/dev/minimal.yaml'))
```

Comment:
We should use the existing values.yaml instead of creating another file. Otherwise we need to keep those in sync and people may be testing things which differs from values.yaml

Reply:
any reason not to use the github team? the nice part of that is you can ping the team in one go

Reply:
Understood. I'll update this is remove the extra values.yaml. I personally found that a reduced set of values was helpful in my own testing but I can understand not wanting to maintain another thing. I'll keep that maintenance burden on my own fork. 😉